hack/update-all.sh

```diff
@@ -34,12 +34,12 @@ GOPKG_LOCK_FILE=${PROJECT_DIR}/Gopkg.lock
 
 cd ${PROJECT_DIR}
 
-# Remove possible stale import in generated code
-rm -rf ${CODEGEN_DIR} ${CODEGEN_FILE} ${GOPKG_LOCK_FILE}
-
 # Update dependent package for non-generated code, such as the code-generator itself
 ${BASH_DIR}/update-dep.sh
 
```

Comment:
Why move this?  We should first clean up generated files, then update dependent package.

Reply:
```
github.com/microsoft/frameworkcontroller/pkg/barrier imports
        github.com/microsoft/frameworkcontroller/pkg/client/clientset/versioned: no required module provides package github.com/microsoft/frameworkcontroller/pkg/client/clientset/versioned; to add it:
        go get github.com/microsoft/frameworkcontroller/pkg/client/clientset/versioned
github.com/microsoft/frameworkcontroller/pkg/controller imports
        github.com/microsoft/frameworkcontroller/pkg/client/informers/externalversions: no required module provides package github.com/microsoft/frameworkcontroller/pkg/client/informers/externalversions; to add it:
        go get github.com/microsoft/frameworkcontroller/pkg/client/informers/externalversions
github.com/microsoft/frameworkcontroller/pkg/controller imports
        github.com/microsoft/frameworkcontroller/pkg/client/listers/frameworkcontroller/v1: no required module provides package github.com/microsoft/frameworkcontroller/pkg/client/listers/frameworkcontroller/v1; to add it:
        go get github.com/microsoft/frameworkcontroller/pkg/client/listers/frameworkcontroller/v1
```